src/example/webChat/client/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "example-service-desk-webchat-client",
+  "version": "1.0.0",
+  "description": "Webchat client service desk example.",
+  "main": "src/buildEntry.ts",
+  "browserslist": [
+    "> 0.25%",
+    "last 2 versions",
+    "ie 11",
```

Comment:
We should make a note to change these to `not ie 11` at some point post 5.0 release.